lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -3786,6 +3923,7 @@ class DecomposeComplexOpsPass
     addPatternIfTargetOpIsIllegal<DecomposeAtenLeakyReluOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenLeakyReluBackwardOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenNewEmptyStridedOp>(patterns);
+    addPatternIfTargetOpIsIllegal<DecomposeAtenBucketizeTensorOp>(patterns);
```

Comment:
You need to also add the op here: https://github.com/llvm/torch-mlir/blob/95361747c2c350886d192261667bfc57d870971d/lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp#L335